### PR TITLE
Created Register and Login pages with validations, added redirection (navigation) to the home page

### PR DESCRIPTION
**Pull request by iasssy**

### CODE REVIEW

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import Typography from '@mui/material/Typography';
+import { Stack, TextField, Button } from '@mui/material';
+import axios from 'axios';
```
Comment (hdcola) [inline src/pages/Login.tsx:4]: maybe you can try react-query

File: src/pages/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                </Typography>
+                {error && (
+                    <Typography color='error'>
+                        {Array.isArray(error) ? error.join(', ') : error}
+                    </Typography>
+                )}
+                {success && (
+                    <Typography color='success.main'>{success}</Typography>
+                )}
+                <TextField
```
Comment (hdcola) [inline src/pages/Login.tsx:84]: <TextField
          error
          id="outlined-error-helper-text"
          label="Error"
          defaultValue="Hello World"
          helperText="Incorrect entry."
        />
        
        https://mui.com/material-ui/react-text-field/#validation
        
        You can use error and helperText to show validation information in a field.

**Review by hdcola — COMMENTED** [review]
You can try using useQuery if you wish.

You can also use the error and helperText properties of TextFields to display validation messages.

File: src/pages/Login.tsx (hunk 2 above)
Comment (iasssy) [reply src/pages/Login.tsx:84]: Thank you, I used error and helperText

File: src/pages/Login.tsx (hunk 1 above)
Comment (iasssy) [reply src/pages/Login.tsx:4]: I couldn't use react-query, just axios. Is it ok?

Comment (hdcola) [reply src/pages/Login.tsx:4]: I will change to  use react-quer next version

**Review by hdcola — APPROVED** [review]

**Pull request merged**